tests/meta-mender-ci/recipes-testing/logger-update-module/logger-update-module.bb

```
@@ -0,0 +1,30 @@
+LIC_FILES_CHKSUM = "file://${COREBASE}/meta/files/common-licenses/Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
```

This is equivalent to:
```suggestion
LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
```

Which we are already using in other recipes